repositories.yaml

```diff
@@ -57,4 +57,35 @@ lib-crash:
     - 'components/lib/crash/metrics.yaml'
   library_names:
     - org.mozilla.components:lib-crash
-  
+engine-gecko:
+  app_id: engine-gecko
+  notification_emails:
+    - frank@mozilla.com
```

Comment:
What are the notification emails used for in the probe-scraper? For general contacts? If so, I think that `android-components-team@mozilla.com` and `geckoview-team@mozilla.com` should be there, in addition to you two ;)